store.ts

```diff
@@ -0,0 +1,36 @@
+import { hookstate, State, useHookstate } from "@hookstate/core";
+import {
+    FirstRunValues,
+    ProfileFormValues,
+    SettingsFormValues,
+} from "./customTypes";
+
+const globalState = hookstate<FirstRunValues>({
+    businessName: "",
+    availableDays: [],
+    deposit: "",
+    operatingHoursStart: "",
+    operatingHoursEnd: "",
+    clientEmailNotifications: false,
+    clientSMSNotifications: false,
+    name: "",
+    email: "",
+    phone: "",
+    password: "",
+    confirmPassword: "",
+});
+
+const wrapState = (state: State<FirstRunValues>) => ({
+    get: () => state.value,
+    setUser: (user: FirstRunValues) => state.set(user),
+    setUserProfile: (values: ProfileFormValues) =>
```

Comment:
you could probably write a `getUserProfile` and `getUserSettings` that return just the values that are in those forms, but also meh